clients/packages/sdk/generate

```diff
@@ -13,7 +13,7 @@ docker rm generate-api || true
 docker run \
     --volume "${SCRIPTPATH}":/sdk \
     --name generate-api \
-    openapitools/openapi-generator-cli:v7.5.0 \
+    openapitools/openapi-generator-cli:v7.4.0 \
```

Comment:
Required since v7.5.0 created some buggy types. Need to investigate later & separately to upgrade once that bug has been resolved.